src/app/dashboard/executions/executiontable/executiontable.component.html

```diff
@@ -39,20 +39,13 @@
   <div class="progress">
     <div class="progress-status">
       <span class="progress-percentage"
-        >{{ 'step' | translate }} {{ execution.currentPluginIndex }}/{{
+        >{{ 'step' | translate }} {{ execution.currentPluginIndex + 1 }}/{{
```

Comment:
`npm run template-check` fails on this line:

    ERROR in src/app/dashboard/executions/executiontable/executiontable.component.html(42,10): : Object is possibly 'undefined'.

`execution.currentPluginIndex` might be undefined, so you should do something like `(execution.currentPluginIndex || 0) + 1` or `execution.currentPluginIndex ? execution.currentPluginIndex + 1 : 1`.